packages/http-client-js/src/components/operation-parameters.tsx

```diff
@@ -19,8 +19,8 @@ export function getOperationParameters(
 ): Record<string, ts.ParameterDescriptor | ay.Children> {
   const transformNamer = useTransformNamePolicy();
   const requiredParameters = operation.parameters.properties
-    .filter((p) => !p.property.optional && !hasDefaultValue(p))
-    .filter((p) => p.path.length === 1);
+    .filter((p) => p.kind !== "bodyProperty" || (p.kind === "bodyProperty" && p.path.length === 1))
```

Comment:
@timotheeguerin do you have any recommendations on how to filter out the body payload properties if the model is not spread ?